From: Jason Dillaman <dillaman@redhat.com>
Date: Wed, 20 Jul 2016 09:15:26 -0400
Subject: journal: improve debug log messages

rbd-mirror debugging involved potentially thousands of journals
concurrently running. The instance address will correlate log
messages between journals.

Signed-off-by: Jason Dillaman <dillaman@redhat.com>
(cherry picked from commit 11475f4fe740cccdfea459ebeabdca8cb94dc911)

Resolves: rhbz#1356064
---
 src/journal/Entry.cc           | 2 +-
 src/journal/JournalMetadata.cc | 2 +-
 src/journal/JournalPlayer.cc   | 2 +-
 src/journal/JournalRecorder.cc | 2 +-
 src/journal/JournalTrimmer.cc  | 2 +-
 src/journal/Journaler.cc       | 2 +-
 src/journal/ObjectPlayer.cc    | 2 +-
 src/journal/ObjectRecorder.cc  | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/journal/Entry.cc b/src/journal/Entry.cc
index f88dea8..257fa58 100644
--- a/src/journal/Entry.cc
+++ b/src/journal/Entry.cc
@@ -9,7 +9,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "Entry: "
+#define dout_prefix *_dout << "Entry: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/JournalMetadata.cc b/src/journal/JournalMetadata.cc
index 63c67f2..de46bd7 100644
--- a/src/journal/JournalMetadata.cc
+++ b/src/journal/JournalMetadata.cc
@@ -11,7 +11,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "JournalMetadata: "
+#define dout_prefix *_dout << "JournalMetadata: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/JournalPlayer.cc b/src/journal/JournalPlayer.cc
index 30d6cdd..750f936 100644
--- a/src/journal/JournalPlayer.cc
+++ b/src/journal/JournalPlayer.cc
@@ -8,7 +8,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "JournalPlayer: "
+#define dout_prefix *_dout << "JournalPlayer: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/JournalRecorder.cc b/src/journal/JournalRecorder.cc
index b4da4ff..4cbe739 100644
--- a/src/journal/JournalRecorder.cc
+++ b/src/journal/JournalRecorder.cc
@@ -8,7 +8,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "JournalRecorder: "
+#define dout_prefix *_dout << "JournalRecorder: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/JournalTrimmer.cc b/src/journal/JournalTrimmer.cc
index a4a47fa..aef16c2 100644
--- a/src/journal/JournalTrimmer.cc
+++ b/src/journal/JournalTrimmer.cc
@@ -9,7 +9,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "JournalTrimmer: "
+#define dout_prefix *_dout << "JournalTrimmer: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/Journaler.cc b/src/journal/Journaler.cc
index 27cc338..c08a11b 100644
--- a/src/journal/Journaler.cc
+++ b/src/journal/Journaler.cc
@@ -19,7 +19,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "Journaler: "
+#define dout_prefix *_dout << "Journaler: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/ObjectPlayer.cc b/src/journal/ObjectPlayer.cc
index 31939e9..43adb5d 100644
--- a/src/journal/ObjectPlayer.cc
+++ b/src/journal/ObjectPlayer.cc
@@ -8,7 +8,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "ObjectPlayer: "
+#define dout_prefix *_dout << "ObjectPlayer: " << this << " "
 
 namespace journal {
 
diff --git a/src/journal/ObjectRecorder.cc b/src/journal/ObjectRecorder.cc
index 5972d89..0cf2fd1 100644
--- a/src/journal/ObjectRecorder.cc
+++ b/src/journal/ObjectRecorder.cc
@@ -10,7 +10,7 @@
 
 #define dout_subsys ceph_subsys_journaler
 #undef dout_prefix
-#define dout_prefix *_dout << "ObjectRecorder: "
+#define dout_prefix *_dout << "ObjectRecorder: " << this << " "
 
 using namespace cls::journal;
 
